staff/victor-abadia/app-template/home.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    
+</body>
```

Comment:
You can add a header to the body with anchor´s to navigate, and a footer with de date of actualization, for example. And in the body a <h1> and a <p> with a description of the page.

Reply:
jaja, the chat interprets the html code, I said You can add a header to the body with anchor´s to navigate, and a footer with de date of actualization, for example. And in the body a "h1" and a "p" with a description of the page